plugins/cilium-cni/cni-install.sh

```diff
@@ -6,12 +6,17 @@ HOST_PREFIX=${HOST_PREFIX:-/host}
 CNI_CONF_NAME=${CNI_CONF_NAME:-10-cilium.conf}
 MTU=${MTU:-1450}
 
-CILIUM_CNI=${CILIUM_CNI:-${HOST_PREFIX}/opt/cni/bin/cilium-cni}
+CNI_DIR=${CNI_BINDIR:-${HOST_PREFIX}/opt/cni}
 CILIUM_CNI_CONF=${CILIUM_CNI_CONF:-${HOST_PREFIX}/etc/cni/net.d/${CNI_CONF_NAME}}
 
-# Install cilium-cni binary tohost
-echo "Installing $CILIUM_CNI ..."
-cp /opt/cni/bin/cilium-cni ${CILIUM_CNI}
+echo "Installing CNI base..."
+wget -nv https://storage.googleapis.com/kubernetes-release/network-plugins/cni-07a8a28637e97b22eb8dfe710eeae1344f69d16e.tar.gz
```

Comment:
The latest one is `network-plugins/cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz`

There's also a amd64 version `network-plugins/cni-amd64-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz`